demos/cat-api/Dockerfile

```diff
@@ -1,10 +1,9 @@
-# use a mminimalist image as root
-FROM openjdk:15-jdk-alpine
+FROM eclipse-temurin:16-jdk
 
 # copy script to load docker-secrets as environment variables and project archive
 ADD src/main/resources/load_docker-secrets_as_env-var.sh load_secrets.sh
 RUN chmod +x load_secrets.sh
 ADD target/api.jar app.jar
 
 # deploy project
-CMD ["/bin/sh", "-c", ". load_secrets.sh && java -jar /app.jar"]
\ No newline at end of file
+CMD ["/bin/bash", "-c", ". load_secrets.sh && java -jar /app.jar"]
```

Comment:
question(non-blocking): Shouldn't this same modification be applied to the other Dockerfiles?

Reply:
The default shell in the suggested image is dash: load_secrets.sh is compatible with both dash and bash, so we don't have any issue right now.

However, it would be a better practice to indeed replace /bin/sh with /bin/bash everywhere, to ensure that shall the docker image change someday, an incompatible shell won't be used. 

I'll fix as suggested.